jvm-packages/dev/change_version.sh

```diff
@@ -19,9 +15,9 @@
 # limitations under the License.
 #
 # (Yizhi) This is mainly inspired by the script in apache/spark.
-# I did some modificaiton to get it with our project.
+# I did some modifications to get it with our project.
```

Comment:
@CodingCat should I leave the spelling mistake for benefit of git history?

Reply:
fix typos is fine